src/widget/textbox.rs

```diff
@@ -144,18 +289,102 @@ impl Widget<String> for TextBox {
         match event {
             Event::MouseDown(_) => {
                 ctx.request_focus();
+                // TODO: hit test and do this for real
+                self.cursor_to(self.selection.end);
                 ctx.invalidate();
+                self.reset_cursor_blink(ctx);
             }
             Event::MouseMoved(_) => {
                 ctx.set_cursor(&Cursor::IBeam);
             }
+            Event::Timer(id) => {
+                if *id == self.cursor_timer {
+                    self.cursor_on = !self.cursor_on;
+                    ctx.invalidate();
+                    let deadline = Instant::now() + Duration::from_millis(500);
+                    self.cursor_timer = ctx.request_timer(deadline);
+                }
+            }
             Event::KeyDown(key_event) => {
                 match key_event {
+                    // Copy (Ctrl+C || Cmd+C)
+                    event
+                        if (event.mods.meta || event.mods.ctrl)
```

Comment:
Note that `druid_shell::keycodes::command_modifier()` tells you which modifier is the command modifier for the platform. It's not available in easy-to-use form though. Also see #81 for related ideas.

This question of keybindings etc needs attention (including the fact that a lot of these are likely best expressed as shortcuts available on an "Edit" menu), but I'm happy getting something useful in early.

Reply:
I couldn't figure out how to compare a `KeyModifier` with a `Modifier`. Would it be okay to do like this?

```rust
const fn command_modifier() -> KeyModifiers {
    #[cfg(target_os = "macos")]
    {
        KeyModifiers {
            shift: false,
            alt: false,
            ctrl: false,
            meta: true,
        }
}
    #[cfg(not(target_os = "macos"))]
    {
        KeyModifiers {
            shift: false,
            alt: false,
            ctrl: false,
            meta: true,
        }
    }
}
```

Reply:
mm, this is interesting.

I think that the correct way to handle this is to have some sort of `CommandList` struct where you register the commands you're looking for, which can have per-platform overrides, and then you pass events to that struct to determine if a command has occurred, but I'll need to think about that a bit more.

For the time being, I think adding `fn contains_sys_command_modifier(self) -> bool` to `KeyModifier` is... okay. 🤢